website/docs/cloud-docs/api-docs/audit-trails-tokens.mdx

```diff
@@ -0,0 +1,147 @@
+---
+page_title: Audit Trails Tokens - API Docs - HCP Terraform
```

Comment:
This is an ongoing suggestion: but to make the plural-singular agree here, we need to either have "trails" or "tokens" be plural, so I suggest keeping "trails" singular for the title:
```suggestion
page_title: Audit Trail Tokens - API Docs - HCP Terraform
```